src/replays/loadFile.ts

```diff
@@ -23,8 +23,14 @@ export async function loadFile(fullPath: string): Promise<FileResult> {
     startTime: null,
     lastFrame: null,
     metadata: null,
+    winners: null,
   };
 
+  const winners = game.getWinners();
```

Comment:
```suggestion
  const winners = game.getWinners();
```
this should always exist or at least be an empty array if unsupported so you don't need the following check.